open_academy/report/session_report.xml

```diff
@@ -0,0 +1,35 @@
+<odoo>
+    <record id="report_session" model="ir.actions.report">
+        <field name="name">Session Report</field>
+        <field name="model">session</field>
+        <field name="report_type">qweb-pdf</field>
+        <field name="report_name">open_academy.session_view_report</field>
+        <field name="report_file">open_academy.session_view_report</field>
+        <field name="attachment_use" eval="True"/>
+        <field name="binding_model_id" ref="model_session"/>
+        <field name="binding_type">report</field>
+    </record>
+
+    <template id="session_view_report">
+        <t t-call="web.html_container">
+            <t t-foreach="docs" t-as="session">
+                <t t-call="web.external_layout">
+                    <div class="page">
+                        <h2 t-field="session.name" />
```

Comment:
You have a lot of `h2` tags, change that

Only session name it's ok with h2.

Check the inventory reports and do something similar,

Session.name with h2

start date in a horizontal table

Attendees inside a vertical table with the field name and email